providers/implementations/signature/build.info

```diff
@@ -2,12 +2,10 @@
 # switch each to the Legacy provider when needed.
 
 $DSA_GOAL=../../libimplementations.a
-$RSA_GOAL=../../libimplementations.a
 
 IF[{- !$disabled{dsa} -}]
   SOURCE[$DSA_GOAL]=dsa.c
 ENDIF
 
-SOURCE[$RSA_GOAL]=rsa.c
-
-
+SOURCE[../../libfips.a]=rsa.c
+SOURCE[../../libnonfips.a]=rsa.c
```

Comment:
Hmmmm, I know why, and it's not the cleanest...  some of the signing code could be placed in crypto/rsa/, especially considering how much is shared between crypto/rsa/rsa_pmeth.c and providers/implementations/signature/rsa.c...

I'm making a mental note on this for the moment being...